stage1/common/types/pod.go

```diff
@@ -90,5 +143,9 @@ func LoadPod(root string, uuid *types.UUID) (*Pod, error) {
 		p.Images[app.Name.String()] = im
 	}
 
+	if err := p.UidRange.Deserialize([]byte(p.PrivateUsers)); err != nil {
```

Comment:
probably anonymously embedding this could cause confusion and it might be better to explicitly name it